deltachat-ios/Controller/WelcomeViewController.swift

```diff
@@ -128,6 +135,7 @@ class WelcomeViewController: UIViewController, ProgressAlertHandler {
         if dcAccounts.getSelected().isConfigured() {
             UserDefaults.standard.setValue(dcAccounts.getSelected().id, forKey: Constants.Keys.lastSelectedAccountKey)
 
+            //FIXME: what do we want to do with QR-Code created accounts? For now: adding an unencrypted account
```

Comment:
it is not unlikely, that, if things work out, encrypted accounts just get the default - then also QR-Code created accounts will get encrypted.